src/services/AuthService.ts

```diff
@@ -0,0 +1,22 @@
+import axios, {AxiosResponse} from "axios";
+import type { LoginRequest } from "../models/LoginRequest";
+import { pbkdf2Sync } from "crypto";
+import { validateLogin } from "../validators/LoginValidator";
+
+export const getToken = async (loginRequest: LoginRequest): Promise<string> => {
+    try {
+        validateLogin(loginRequest.email, loginRequest.password);
+        const hashedPwd: Buffer = pbkdf2Sync(loginRequest.password, "", 65536, 6*16, 'sha256');
+        loginRequest.password = hashedPwd.toString('base64');
+        const response: AxiosResponse = await axios.post("http://localhost:8080/api/auth/login", loginRequest);
```

Comment:
Change this due to the front-end Docker ticket. No hard-coded URLs please.

Reply:
This has been corrected and committed.